documentation/src/main/resources/pages/ditto/protocol-specification-things-messages.md

```diff
@@ -99,7 +99,7 @@ The response we would get from our coffee machine could look something like this
         "content-type": "text/plain",
         "version": 1,
     },
-    "path": "/inbox/messages/ask",
+    "path": "/outbox/messages/ask",
```

Comment:
This is not required - as for correlation only the corrlation-id is used and no additional policy check is performed.
We have examples were `inbox` changes to `outbox` but I'm not sure that this helps understanding the concept :D

Reply:
ok, removed it